jobs/google-bucket-replicate-job.yaml

```diff
@@ -0,0 +1,48 @@
+# run with 
+# gen3 runjob google-bucket-replicate-job.yaml PROJECT google_project INPUT_BUCKET input_bucket LOG_BUCKET log_bucket
+# Ex. gen3 runjob dcf-dataservice/jobs/google-bucket-replicate-job.yaml PROJECT cdis-test-188416 INPUT_BUCKET data-flow-code LOG_BUCKET data-flow-code
```

Comment:
Should the `INPUT_BUCKET`  be the same as the `LOG_BUCKET` in the example? both are `data-flow-code`

Reply:
That is a bad example. You can use the same bucket or two different ones